css/style.css

```diff
@@ -32,21 +32,19 @@ body {
 footer {
 
   & .menu-port {
+    display: flex;
+    flex-direction: row;
+    padding: 20px;
```

Comment:
Karna udh make display flex, better make gap atau sesuain make justify-content
Biar lebih mudah untuk ngatur tampilan yg responsive
https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-justify-content